docs/docs/spark-configuration.md

```diff
@@ -154,43 +154,51 @@ spark.read
-| lookback        | As per table property | Overrides this table's read.split.planning-lookback                                       |
-| file-open-cost  | As per table property | Overrides this table's read.split.open-file-cost                                          |
-| vectorization-enabled  | As per table property | Overrides this table's read.parquet.vectorization.enabled                                          |
-| batch-size  | As per table property | Overrides this table's read.parquet.vectorization.batch-size                                          |
-| stream-from-timestamp | (none) | A timestamp in milliseconds to stream from; if before the oldest known ancestor snapshot, the oldest will be used |
+Iceberg 1.8.0 and later support setting read options by Spark session configuration `spark.datasource.iceberg.<key>=<value>`
+when using DataFrame to read Iceberg tables, for example: `spark.datasource.iceberg.split-size=512m`, it has lower priority
+than options explicitly passed to DataFrameReader.
+
+| Spark option          | Default               | Description                                                                                                       |
+|-----------------------|-----------------------|-------------------------------------------------------------------------------------------------------------------|
+| snapshot-id           | (latest)              | Snapshot ID of the table snapshot to read                                                                         |
+| as-of-timestamp       | (latest)              | A timestamp in milliseconds; the snapshot used will be the snapshot current at this time.                         |
+| split-size            | As per table property | Overrides this table's read.split.target-size and read.split.metadata-target-size                                 |
+| lookback              | As per table property | Overrides this table's read.split.planning-lookback                                                               |
+| file-open-cost        | As per table property | Overrides this table's read.split.open-file-cost                                                                  |
+| vectorization-enabled | As per table property | Overrides this table's read.parquet.vectorization.enabled                                                         |
+| batch-size            | As per table property | Overrides this table's read.parquet.vectorization.batch-size                                                      |
+| stream-from-timestamp | (none)                | A timestamp in milliseconds to stream from; if before the oldest known ancestor snapshot, the oldest will be used |
 
 ### Write options
 
-Spark write options are passed when configuring the DataFrameWriter, like this:
+Spark write options are passed when configuring the DataFrameWriterV2, like this:
```

Comment:
I replaced the example with `DataFrameWriterV2` because

https://github.com/apache/iceberg/blob/319f29ea860e42e7cc21cda8c05d882134e6431f/docs/docs/spark-writes.md#L264